crates/rv/src/commands/ci.rs

```diff
@@ -110,7 +120,7 @@ async fn ci_inner(cache: &rv_cache::Cache, args: &CiInnerArgs) -> Result<()> {
     let lockfile_contents = tokio::fs::read_to_string(&args.lockfile_path).await?;
     let lockfile = rv_lockfile::parse(&lockfile_contents)?;
     let gems = download_gems(lockfile, cache, args).await?;
-    install_gems(gems, args).await?;
+    install_gems(gems, args, &args.ruby_version).await?;
```

Comment:
What's the reason behind pulling the version out of the `args` here, versus letting `install_gems` pull it from `args` internally?

Reply:
No reason, that's a silly mistake from me!